src/helpers/getAccessToken.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+import { parse as parseCookie } from 'cookie'
+import { AUTH_COOKIE } from '@constants/cookies'
+
+const getAccessToken = (context?: any): string => {
+  const cookies =
+    context && context.ctx.req
```

Comment:
this `context.ctx` is weird… in the docs it’s usually `context.req`… I think they changed something but didn’t update the docs yet.

Reply:
🤷‍♂️